\chapter{Introduction}
\label{chap:intro}

% Check Reyka and Bailey's introduction

% Intro: slightly more than 30 pages in a review form, and set up framework (topics and challenges) for the projects
% Discussion: remaining challenges, the benefits in another fields than oncology

% Cancer
% mutation
% Multi-omics

% PTM
% heterogeneity
% tumor microenvironment
% precision medicine


% Large scale patient samples collection (consoritum works)

% data portal

% data sharing
% pipeline execution
% gene annotation
% integration


% GBM

% Outline of the dissertation


\section{Cancer and precision medicine}
Cancer is the leading cause of death in most countries in the world \cite{sungh_brayf:GlobalCancer2021}. In the U.S., cancer is estimated to introduce roughly 1.9 millions cases and 600 thousands deaths in 2021, where \textasciitilde40\% of the population will be diagnosed with cancer at some point in their lifetime \cite{siegelrl_jemala:CancerStatistics2021}. Despite advancements in understanding and treating cancer in the past 50 years, many forms of the disease lack effective treatment, and how normal cells become cancerous and form a tumor, the process known as oncogenesis, remains to be fully deciphered. Therefore, the cancer research community continues to work on better characterization of cancer and its treatment as our ultimate goals.

The functional properties of a tumor have been described as ``cancer hallmarks'' \cite{hanahand_weinbergra:HallmarksCancer2011}, including cell proliferation, immortality in replication, cell death evasion, and increased ability of angiogenesis and invasion. While the tumor microenvironment, the surrounding tissue environment where a tumor develops, consists of normal cells, the tumor microenvironment also helps maintain tumor abbrent growth and immune respones beneficial to the tumors \cite{hanahand_weinbergra:HallmarksCancer2011,quaildf_joyceja:MicroenvironmentalRegulation2013}. To develop an effective cancer treatment, it's important to understand how cancer hallmarks are achieved during the oncogenesis and the complicated cell-cell interactions in the tumor microenvironment.


\subsection{Oncogenesis and the functional impact of somatic mutation}
Oncogenesis is currently viewed as a microevolution process where normal cells acquire somatic mutations that offer growth and survival advantage \cite{strattonmr_futrealpa:CancerGenome2009,martincorenai_campbellpj:SomaticMutation2015}. The somatic mutation rate in human is estimated to be about 2 to 10 mutations per cell division \cite{lynchm_lynchm:RateMolecular2010,milhollandb_vijgj:DifferencesGermline2017}. A tumor may accumulate 0.01--100 mutations per megabase in the genome depending on the cancer type \cite{lawrencems_getzg:MutationalHeterogeneity2013,martincorenai_campbellpj:SomaticMutation2015}. However, only some of the mutations might confer a growth and survival advantage while remaining mutations are either benign or of unknown phenotype. Such actively selected mutations are frequently found in certain genes, known as cancer driver genes and significantly mutated genes (SMGs) \cite{dingl_mariamidzea:PerspectiveOncogenic2018}. There are three main types of cancer driver genes: proto-oncogenes that promotes cell growth, tumor suppressor genes that regulate cell growth, and DNA rapair genes that fix DNA damages.

Mutations in one cancer gene or a combination of them are sufficient to drive the oncogenic process. For example, transcription factor \gene{TP53} is a tumor suppressor gene\footnotemark{} and it responds to DNA damage by inducing cell cycle arrest \cite{kastanmb_craigrw:ParticipationP531991}. Mutations in TP53 can lead to the uncontrolled proliferation of mutated cells, thereby causing cancer. Thus, \gene{TP53} is one of the most commonly found cancer genes in a tumor, whose mutations can be found in a remarkable 37.5\% of over 9 thousands tumor samples examined across 33 cancer types \cite{baileymh_dingl:ComprehensiveCharacterization2018}. On the other hand, \gene{EGFR}, epidermal growth factor receptor, is a well known proto-oncogene that encodes a transmembrane protein that activates cell signaling pathways to promote cell growth. The mutations in \gene{EGFR} often lead to increased gene expression or uncontrolled activation of its function, and are commonly found in breast, lung, and brain cancer patients \cite{ciardiellof_tortorag:EGFRAntagonists2008}.

\footnotetext{\gene{TP53}'s role as a cancer drive gene is complicated. While the majority of its mutations are loss-of-function, leading to decreased ability to suppress tumor growth, it also exhibits oncogene properties with gain-of-function mutations \cite{petitjeana_olivierm:TP53Mutations2007}.}

Therefore, characterizing the functional impact of a mutation has been an area of intensive research ever since the first cancer somatic mutation was identified in \citeyear{reddyep_barbacidm:PointMutation1982} \autocite{reddyep_barbacidm:PointMutation1982,tabincj_changeh:MechanismActivation1982}. Besides well controlled experimental validation, a number of statistical methods have been widely-used to infer functional elements, including high gene mutation rate, mutation recurrence, mutual exclusivity, and co-occurrence of mutations across multiple samples or cancer types, since these phenomena imply positive selection \cite{martincorenai_campbellpj:SomaticMutation2015}. However, only a few mutations are well understood and the majority of them have unknown function in cancer. Even for well-known cancer driver genes like \gene{PIK3CA} and \gene{BRCA1/2}, only a fraction of suspected cancer-related mutations having actually been functionally validated \cite{ngpks_millsgb:SystematicFunctional2018}. It is still challenging today to attribute a phenotypic ``cancer hallmark'' expressed by a tumor to its mutations.


\subsection{Multi-omics portrait of cancer}
Since the phenotypes of cancer hallmarks are achieved through abberant alterations at multiple molecular levels, a new paradigm to understand complex diseases like cancer is through the lens of multi-omics analysis \cite{deanda-jaureguig_hernandez-lemuse:ComputationalOncology2020}. Thanks to the recent advancements in high-throughput experiments (commonly termed ``omics''), we are able to profile the specimen systematically at a molecular level. Common omics assays include:

\tightlists
\begin{itemize}
    \tightlist
    \item Genomic: whole exome sequencing, whole genome sequencing
    \item Transcriptome: RNA sequencing, miRNA sequencing
    \item Epigenomics: DNA methylation microarray, whole genome bisulfite sequencing, ChIP-seq, ATAC-seq
    \item Proteomics: TMT multiplexed mass spectrometry
    \item Single cell omics: single cell/nuclei RNA and ATAC sequencing
    \item Imaging: Imaging mass cytometry, CODEX, histopathology H\&E slides
    \item Spatial transcriptomics: 10x Genomics Visium
\end{itemize}

By investigating the biological signals and features accquired from different molecular assays in the same specimen, we are able to ``connect the dots'' and reason how a genetic alteration contributed to the phenotypes, furthermore, how a set of genetic alterations interact together. For example, \gene{EGFR} is an oncogene commonly altered in glioblastoma \cite{eskilssone_miletich:EGFRHeterogeneity2018}, but we are unsure of its functional impace. Using multi-omics approach, we will be more confident to attribute the cell profileration of a glioblastoma to genetic alterations in \gene{EGFR} if we are able to pick up consistent signs from different experiment assays: mutation or amplification of \gene{EGFR} is detected; RNA expression, protein abundance, and activating phosphorylation of EGFR are all increased in this sample relative to the wild-type tumors and normal samples; increased activity in the RTK-RAS-MAPK pathway; single cell and imaging omics showing the increased EGFR activity is found in tumor cells and not normal cells in the tumor microenvironment.

Large scale data studies \cite{hutterc_zenklusenjc:CancerGenome2018,rodriguezh_lowydr:NextHorizon2021}


% Due to the random nature of oncogenesis, each tumor is different in their mutational profile.


% Our lab has also studied the effect of coding mutations on protein structure \autocite{niu_protein-structure-guided_2016}. \citeauthor{niu_protein-structure-guided_2016} found that mutations cluster spatially in the functional protein domains of driver genes. For example, mutations clustering near an EGFR phosphotyrosine site in the 3D tertiary structure have been shown to increase autophosphorylation levels and EGFR activity \autocite{niu_protein-structure-guided_2016}.

% Many somatic mutations have unknown significance; transition to PTM
%  Since post-translational modifications (PTMs) are known to be essential for signal transduction \autocite{hunter_signaling2000_2000}, I propose to explore the functional impacts of mutations by investigating their effects on PTMs.


\subsection{Precision medicine}



\begin{figure}[tb]
    \centering
    \includegraphics[width=1\linewidth]{figures/chap01_intro/cptac_gbm_multi-omics.png}
    \caption[Overview of the CPTAC GBM data collection and study design.]{%
        Overview of the CPTAC GBM data collection and study design.
        \sourceatright[2em]{\footnotesize Created with BioRender.com}
    }
    \label{fig:intro-cptac-gbm-study design}
\end{figure}


