\chapter{PTMcosmos}
\label{chap:ptmcosmos}



\section{Summary}
PTMcosmos is a comprehensive database with an interactive web portal designed to catalog and visualize post-translational modifications (PTMs) in humans. It contains 469,183 experimentally-validated PTM sites and their supporting evidence from UniProt Knowledge Base, PhosphoSitePlus, and Clinical Proteomic Tumor Analysis Consortium (CPTAC). PTMcosmos summarizes the entire spectrum of CPTAC proteomics data on human cancer patients, including protein and PTM peptide abundance data from 10 different cancer types. Additionally, PTMcosmos contains cancer somatic mutations from the Cancer Genome Atlas (TCGA), thus allowing for the collective integration and analysis of different data types.  In PTMcosmos, we have built an ensemble of interactive visualization tools that will allow researchers to investigate altered PTM functions due to genetic alterations in close proximity. The database is live at \url{https://ptmcosmos.wustl.edu}. Furthermore, we used PTMcosmos to investigate PTM regulation across cancer types. First, we investigated the differential abundance of the PTM sites of cancer driver genes, focusing primarily on phosphorylation of the tumor suppressor retinoblastoma protein (encoded by \gene{RB1})  and acetylation of the histone acetyltransferase E1A Binding Protein P300 (\gene{EP300}) across cancer types. We analyzed the association between these PTM events and downstream targets, as well as with tumor subtypes, significantly mutated gene (SMG) mutation status  and clinical features. Second, we investigated the association of the protein abundance of cancer driver genes with ubiquitylsites in lung squamous cell carcinoma (LSCC) to nominate potentially novel modes of regulation of these proteins’ activity. We further analyzed the tumor subtype specificity and tumor-normal abundance changes of these ubiquitylsites and their corresponding substrate proteins, identifying several EGFR ubiquitylsites which may regulate EGFR abundance in LSCC. Finally, we identified the linear and spatial clustering of mutations and PTM sites, identifying multiple mutation-PTM clusters in cancer related genes, including TP53, PIK3CA, CTNNB1, EGFR, and IDH1. We envision that PTMcosmos will serve both the CPTAC consortium and the wider research community to better understand the role of PTMs in cancer.


\section{Introduction}
Post-translational modifications (PTMs) act as ``switches'' of protein activity and have been shown to be commonly dysregulated in cancer, controlling the pathway activities and tumor phenotypes \cite{hanahand_weinbergra:HallmarksCancer2011}. Phosphorylation \cite{severr_bruggejs:SignalTransduction2015}, acetylation \cite{dimartilem_trisciuogliod:MultifacetedRole2016,gilj_encarnacion-guevaras:LysineAcetylation2017}, and ubiquitination \cite{dengl_wangp:RoleUbiquitination2020} have all be shown to play distinct roles in tumorigenesis. The known function of the detected PTM sites are curated through a long term effect by databases such as UniProt \cite{theuniprotconsortium_theuniprotconsortium:UniProtWorldwide2019} and PhosphoSitePlus \cite{hornbeckpv_skrzypeke:PhosphoSitePlus2015} by recording the reported publications and validation experiments. PTM dysregulation through mutations in cancer has been actively explored, where linearly and spatially closed mutations could disrupt the PTM site and affect the underlying protein function \cite{reimandj_badergd:MutationalLandscape2013,creixellp_lindingr:KinomewideDecoding2015,wagiho_badergd:MIMPPredicting2015,krassowskim_reimandj:ActiveDriverDBHuman2018,huangk_dingl:SpatiallyInteracting2021}.

Recent advancements in high-throughput multiplexed mass spectrometry have enabled consortia such as Clinical Proteomic Tumor Analysis Consortium (CPTAC) \cite{%
zhangh_townsendrr:IntegratedProteogenomic2016,
rodriguezh_penningtonsr:RevolutionizingPrecision2018,
clarkdj_zhangh:IntegratedProteogenomic2019,
vasaikars_clinicalproteomictumoranalysisconsortium:ProteogenomicAnalysis2019,
douy_zhaog:CPTACUCEC2020,
mcdermottje_rodlandkd:ProteogenomicCharacterization2020,
krugk_zimmermanlj:ProteogenomicLandscape2020,
huy_shiz:IntegratedProteomic2020,
petraliaf_bocikwe:IntegratedProteogenomic2020,
gillettema_shiz:ProteogenomicCharacterization2020,
huangc_zhuj:ProteogenomicInsights2021,
wanglb_cptac:GBM2021,
rodriguezh_lowydr:NextHorizon2021}
and International Cancer Proteogenome Consortium (ICPC) \cite{%
gaoq_fanj:IntegratedProteogenomic2019,
mundg_hwangd:ProteogenomicCharacterization2019,
chenyj_chenyj:ProteogenomicsNonsmoking2020}
to capture tens of thousands of PTM sites across a large number of tumor samples of multiple cancer types. However, several challenges exist to impede a researcher's ability to utilize the large-scale datasets and investigate the PTM sites of interest. It is difficult to integrate the datasets with existing PTM databases due to the annotation differences. The protein sequencing differences used by different projects result in the coordinate change of PTM sites. We previously developed a tool to prioritize the phospho mutation interactions spatially close on 3D protein structures \cite{huangk_dingl:SpatiallyInteracting2021}, however, the interactions of mutation and other PTM types have not been explored. Non-trivial bioinformatic expertise is required to carry out the above analyses.

Here we present PTMcosmos, a comprehensive database for the collection, annotation, and visualization of PTM sites in cancer. This database enables computational and experimental researchers to analyze the abundance, detection rates and supporting evidence of PTM sites, as well as their interplay with mutations in linear and 3D space. Additionally, we demonstrate the utility of PTMcosmos in pan-cancer analyses involving PTM dysregulation through abundance changes and interaction with mutations. We anticipate that PTMcosmos will serve the cancer research community and further our understanding of PTMs in cancer.



\section{Results}

\subsection{Data collection and harmonization}
We developed PTMcosmos, a web portal to provide a comprehensive resource to the proteogenomics community to study cancer \fref{fig:ptmcosmos-workflow}. Total 432,021 PTM sites were collected from two databases, UniProt Knowledge Base and PhosphoSitePlus, and two experimental proteomic consortia across 13 studies and 11 cancer types, CPTAC and ICPC \fref{fig:ptmcosmos-stats-cptac-by-cancer}. To study the mutational impact on PTMs, we also collected the somatic mutations from the matching CPTAC tumor samples and a pan-cancer collection from TCGA MC3 project \cite{ellrottk_tcga:MC3MutationCalling2018}. Additional protein annotations were loaded to bridge the proteins with their known biological functions, including the non redundant protein sequence archive from UniPar \cite{leinonenr_apweilerr:UniProtArchive2004}, standardized gene symbol, aliases and identifiers from HUGO Gene Nomenclature Committee (HGNC) \cite{brufordea_tweedies:GuidelinesHuman2020}, protein structures from Protein Data Bank (PDB) \cite{bermanhm_bournepe:ProteinData2000}, protein domains from InterPro \cite{blumm_finnrd:InterProProtein2021} and Pfam \cite{mistryj_batemana:PfamProtein2021}, and protein pathways and complexes from OmniPath \cite{tureid_saez-rodriguezj:OmniPathGuidelines2016}.

\begin{figure}[tb]
    \centering
    \includegraphics[width=0.9\linewidth]{figures/chap03_ptmcosmos/figure1_ptmcosmos_workflow.pdf}
    \caption[PTMcosmos overview.]{PTMcosmos overview.}
    \label{fig:ptmcosmos-workflow}
\end{figure}

To avoid ambiguous PTM location due to multiple protein sequence versions, we keep track of the underlying peptide sequences of the PTM sites and harmonized. We used UniProt Archive (UniParc) \cite{leinonenr_apweilerr:UniProtArchive2004} to check for sequence identity and performed global protein sequence alignment to mapped all the reported PTM sites to UniProt canonical protein isoforms (release 2021.02) (\fref{fig:ptmcosmos-map-stats-schematic}; Methods). For PhosphoSitePlus whose sequences are mostly based on UniProt, over 99.9\% of the PTM sites were reported on the proteins identical to UniProt. Only 0.3\% of them (n = 1,060) changed their coordinates after the mapping and only 0.01\% of them (n = 33) failed to be mapped to UniProt. Across the experiment cohorts where RefSeq protein sequences were used in the data generation (\tref{tab:ptmcosmos-peptide-db}), about 60--75\% of the PTM sites are reported on the identical protein sequences to UniProt, while 10--20\% of them changed their coordinates after the mapping and 1--3\% of them are not mappable (\fref{fig:ptmcosmos-map-stats-acetyl}--\subcaptionref{fig:ptmcosmos-map-stats-phospho}).

\begin{figure}[tbp]
    \centering
    \phantomlabel{fig:ptmcosmos-map-stats-schematic}
    \phantomlabel{fig:ptmcosmos-map-stats-acetyl}
    \phantomlabel{fig:ptmcosmos-map-stats-ubiquityl}
    \phantomlabel{fig:ptmcosmos-map-stats-phospho}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figures1_mapping_stats.pdf}
    \caption[Supporting details of the peptide re-annotation and coordinate mapping.]{%
        Supporting details of the peptide re-annotation and coordinate mapping.
        \subref{fig:ptmcosmos-map-stats-schematic}
        A schematic representation of the peptide coordinate mapping using global protein sequence alignment. Peptides (a--e) including the flanking sequence are the PTM sites to be mapped. Only peptides (a--c) in the yellow regions produced by the sequence alignment are mappable. Peptide d is not mappable since only a parital of its sequence can be aligned. Peptide e is also not mappable due to a mismatch between the source and target protein sequences. PTM peptides mapability of \subref{fig:ptmcosmos-map-stats-acetyl} acetylation,
        \subref{fig:ptmcosmos-map-stats-ubiquityl} ubiquitination,
        \subref{fig:ptmcosmos-map-stats-phospho} phosphorylation CPTAC proteome datasets.
    }
    \label{fig:ptmcosmos-map-stats}
\end{figure}

By combining all the data sources and annotations. a total of 3.1 millions exhibits of PTM supporting evidences are collected from over 75 thousands of publications and other kinds of experimental validations in the database (\fref{fig:ptmcosmos-stats-publications-per-ptm}, \ref{fig:ptmcosmos-stats-publications-per-year}). The majority of PTM sites received 1--5 supporting publications, where the phosphosites are much more investigated and reported than other PTM types. Surprisingly, still a large number (> 20\%) of PTM sites have low to no literature support. Thanks to the massive effort of literature curation by UniProt and PhosphoSitePlus, the collected literature can be dated back to 1980s. Research of earlier days focused on the characterization of small number of PTM sites, whereas many new PTM sites were discovered using the high throughput methods.

\begin{figure}[tbp]
    \centering
    \phantomlabel{fig:ptmcosmos-stats-cptac-by-cancer}
    \phantomlabel{fig:ptmcosmos-stats-publications-per-ptm}
    \phantomlabel{fig:ptmcosmos-stats-publications-per-year}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure1_ptmcosmos_stats.pdf}
    \caption[Overview of experiment proteome datasets and distribution of PTM supporting evidence collected in PTMcosmos.]{%
        Overview of experiment proteome datasets \subref{fig:ptmcosmos-stats-cptac-by-cancer} and distribution of PTM supporting evidence collected in PTMcosmos.
        \subref{fig:ptmcosmos-stats-publications-per-ptm}
        Distribution of supporting publications per PTM type.
        \subref{fig:ptmcosmos-stats-publications-per-year}
        Distribution of supporting publications along its publication time.
    }
    \label{fig:ptmcosmos-stats}
\end{figure}


\subsection{Descriptive analysis of PTM sites in PTMcosmos}
We analyzed the PTM sites in PTMcosmos to understand the pathways and domains present in the database. All 50 cancer hallmark pathways were represented in PTMcosmos, with phosphorylation being the dominant PTM for each pathway (\fref{fig:ptmcosmos-site-detail-pathway-domain}). The mitotic spindle, G2/M checkpoint and E2F pathways displayed the largest number of PTM sites, while pancreas beta cells, Notch signaling and angiogenesis showed the lowest number of PTM sites. To understand these results in greater detail, we investigated the genes and domains represented among PTM sites in the top three pathways. Consistent with their presence in cytoskeletal proteins, spectrin and filamin repeats had the highest number of PTMs (n = 655; 639) in the mitotic spindle pathway, with phosphorylation being the dominant PTM among these domains (n = 283; 390). The RecF/RecN/SMC N terminal domain, found in structural maintenance of chromosomes (SMC) proteins, was highly abundant in both the G2/M checkpoint (n = 342) and E2F target pathways (n =4 68), with ubiquitination being the dominant modification in this domain (n = 165; 194). The protein kinase domain, modified predominantly by ubiquitination and phosphorylation, was the most represented domain among G2/M checkpoint pathway members (151 phospho- and 141 ubiquitylsites out of 343 PTM sites).

\begin{figure}[tbp]
    \centering
    \phantomlabel{fig:ptmcosmos-site-detail-venn}
    \phantomlabel{fig:ptmcosmos-site-detail-pathway-domain}
    \phantomlabel{fig:ptmcosmos-site-detail-phspho-tyr}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure2_ptmcosmos_site_detail.pdf}
    \caption[Descriptive analysis of the PTM sites in PTMcosmos.]{%
        Descriptive analysis of the PTM sites in PTMcosmos.
        \subref{fig:ptmcosmos-site-detail-venn}
        Venn diagram showing unique CPTAC and database sites. Database sites are only found in alternative evidence sources (PhosphoSitePlus and UniProtKB).
        \subref{fig:ptmcosmos-site-detail-pathway-domain}
        The top Hallmark pathways and PFAM domains (Y axis) represented among all PTM sites in PTMcosmos. For each pathway or domain, the number of PTM sites (X axis) is normalized by the total protein sequence length of entries in the given pathway or domain.
        \subref{fig:ptmcosmos-site-detail-phspho-tyr}
        The proportion of tyrosine phosphosites present among all CPTAC sites compared to sites uniquely present in PhosphositePlus or UniProtKB (referred to as ``unique DB'' sites).
    }
    \label{fig:ptmcosmos-site-detail}
\end{figure}

Next, we investigated the PTM sites detected only in CPTAC and those found only in alternative evidence sources (PhosphoSitePlus and UniProtKB). In total, 93,319 PTM sites were unique to CPTAC experiments (hereafter called unique CPTAC sites), while 258,935 sites were detected in PhosphoSitePlus or UniProtKB, but not CPTAC (hereafter called unique database sites) (\fref{fig:ptmcosmos-site-detail-venn}).
Among all the Pfam protein domains tested (n=2,501),  248 domains were significantly enriched among the unique database sites, including ``zinc finger, C2H2 type'' (FDR = 3.84e-147), ``cadherin domain'' (FDR = 3.08e-127), ``core histone H2A/H2B/H3/H4'' (FDR = 7.13e-90), ``immunoglobulin V-set domain'' (FDR = 2.53e-56) and “protein kinase domain” (FDR = 2.33e-53).
Four hallmark pathways were significantly enriched among the unique database sites: ``HALLMARK\_INFLAMMATORY\_RESPONSE'' (FDR = 3.10e-32), ``HALLMARK\_SPERMATOGENESIS'' (FDR = 3.94e-31), ``HALLMARK\_IL6\_JAK\_STAT3\_SIGNALING'' (FDR = 3.01e-10) and ``HALLMARK\_NOTCH\_SIGNALING'' (FDR = 9.83e-05).
Similarly, 75 protein domains were significantly enriched among unique CPTAC sites, with the five most significantly enriched domains being ``spectrin repeat'' (FDR = 1.15e-73), ``lipocalin/cytosolic fatty-acid binding protein family'' (FDR = 7.33e-35), ``serpin (serine protease inhibitor)'' (FDR = 3.04e-23), ``apolipoprotein A1/A4/E domain'' (FDR = 3.31e-20) and ``transferrin'' (FDR = 1.72e-18).
Seven hallmark pathways were significantly enriched among unique CPTAC sites, with the top five pathways being ``HALLMARK\_COAGULATION'' (FDR = 1.05e-83), ``HALLMARK\_EPITHELIAL\_MESENCHYMAL\_TRANSITION'' (FDR = 1.75e-62), ``HALLMARK\_KRAS\_SIGNALING\_DN'' (FDR = 8.38e-29), ``HALLMARK\_COMPLEMENT'' (FDR = 4.55e-19) and ``HALLMARK\_ANGIOGENESIS'' (FDR = 3.28e-13).
Interestingly, among serine, threonine and tyrosine phosphosites, unique database phosphosites were enriched for phosphotyrosine compared to all CPTAC sites (OR = 0.17, p < 2.2e-16) (\fref{fig:ptmcosmos-site-detail-phspho-tyr}), suggesting a potential limitation of mass spectrometry technology in detecting phosphotyrosine.


\subsection{PTMcosmos user interface}
To facilitate a friendly user interface, we made all the information searchable on PTMcosmos in a similar design as UniProt (\fref{fig:ptmcosmos-usage-demo}). Users are able to query the gene and protein name of interest and the PTM information is organized on a protein basis. The page contains a protein feature viewer to show the location of PTM sites and protein domain along its sequence. For each PTM site, its supporting evidence is grouped into dropdown menus based on the evidence source and type. Details of each evidence exhibit is available in the dropdown menu, including the publication information, sequence similarity source, and the experiment cohorts detecting the site. For PTM sites with experiment datasets loaded in PTMcosmos, an additional interactive page is available for users to explore the PTM abundance in the dataset. For proteins with mutation and PTM spatial clusters reported by HotPTM, an interactive viewer is available for users to explore the protein structure annotated with the PTM-mutation clusters.

\begin{figure}[tb]
    \centering
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure3_ptmcosmos_usage.pdf}
    \caption[PTMcosmos usage demonstration.]{%
        PTMcosmos usage demonstration using β catenin phosphosites as an example. From right to left, the phosphosite S33, from the literature, is known to regulate the degradation of itself. The mutational impact analysis by HotPTM shows that endometrial cancer samples frequently harbor nearby mutations that disrupt this phosphosite. Finally, users can quickly check the relative abundance change at peptide level in every published CPTAC cohort.
    }
    \label{fig:ptmcosmos-usage-demo}
\end{figure}


\subsection{Differential expressed PTM sites between tumor and normal samples across cancer types}
To showcase how PTMcosmos can help research investigate PTM's function in cancer,  we examined PTM sites differentially expressed between tumor and normal adjacent tissue samples. To uncover functionally relevant PTM sites, we restricted our analysis to PTMs of 188 known cancer driver genes, 180 of which had PTM sites detected in the CPTAC cohorts \cite{baileymh_dingl:ComprehensiveCharacterization2018}. We found a total of 2,055 differentially expressed PTM sites across 146 driver genes, consisting of 1,405 phosphosites and 522 acetylsites (\fref{fig:ptmcosmos-de-overview}). To further prioritize functional PTM sites, we used PTMcosmos to select for sites detected in previous studies. Several driver genes showed differential acetylation and phosphorylation across cancer types. CDK12 and EZH2 phosphorylation were upregulated across the same six cancer types (UCEC, OV, LUAD, LSCC, GBM, and CRC). Acetylation and phosphorylation of the cell cycle protein nucleophosmin (NPM1) was consistently upregulated in LUAD and LSCC, with the NPM1-T199 phosphosite, known to play a role in the DNA damage response, also had enriched detection in LSCC tumors compared to normal adjacent tissue. The chromatin remodelling protein BRG1 (SMARCA4) also showed increased phosphorylation across multiple cancer types, including LSCC and LUAD. Notably, four cancer types showed upregulated acetylation of the histone acetyltransferase p300, and eight cancer types displayed increased phosphorylation of the canonical tumor suppressor Rb.

\begin{figure}[p]
    \centering
    \phantomlabel{fig:ptmcosmos-de-overview}
    \phantomlabel{fig:ptmcosmos-ep300-prot-paint}
    \phantomlabel{fig:ptmcosmos-ep300-hist-acetyl}
    \phantomlabel{fig:ptmcosmos-ep300-assoc}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure4_ptmcosmos_ep300.pdf}
    \caption[Differential expression analysis of sites in PTMcosmos.]{%
        Differential expression analysis of sites in PTMcosmos.
        \subref{fig:ptmcosmos-de-overview}
        Overview of differentially expressed driver gene acetyl- and phosphosites across 8 cancer types.
        \legendcontdnote
    }
    \label{fig:ptmcosmos-ep300}
\end{figure}
\begin{figure}[t]
    \centering
    \legend{%
        \legendcontdref{fig:ptmcosmos-ep300}
        \subref{fig:ptmcosmos-ep300-prot-paint}
        Differentially expressed acetylsites of the acetyltransferase p300 mapped onto the protein's domain structure. The numbers shown in the circles represent the number of samples in a cohort detecting the given acetylsite.
        \subref{fig:ptmcosmos-ep300-hist-acetyl}
        Heatmap showing the significant Pearson correlations between the aggregate p300 acetylation score and histone acetylsites.
        \subref{fig:ptmcosmos-ep300-assoc}
        Mosaic plot showing the associations between p300 acetylation status and categorical variables such as tumor subtypes and the alteration status of significantly mutated genes.
    }
\end{figure}


\subsection{Tumor-normal comparison of p300 phosphosites}



\subsection{Tumor-normal comparison of Rb1 phosphosites}
We observed the upregulation of multiple Rb phosphosites across 8 cancer types (OV, CRC, LSCC, LUAD, GBM, UCEC, HNSCC, CCRCC) known to disrupt the Rb-E2F interaction, thus promoting E2F transcriptional activity and cell cycle progression \cite{rubinsm_rubinsm:DecipheringRetinoblastoma2013} (\fref{fig:ptmcosmos-rb1-prot-paint}).  Interestingly, phosphorylation of Rb at T356, known to promote an order-to-disorder transition in the RbN domain, was significantly upregulated across six cancer types \cite{rubinsm_rubinsm:DecipheringRetinoblastoma2013}. We found that in CPTAC tumors, OV and HNSCC tumors showed increased Rb phosphorylation at S788, while LSCC and GBM tumors exhibited upregulated phosphorylation at S795, with both sites known to in part disrupt binding between E2F and the C-terminal domain of Rb (RbC) \cite{rubinsm_pavletichnp:StructureRb2005}. Dual phosphorylation of Rb at T821 and T826 was increased in LUAD, LSCC and CRC tumors. Phosphorylation of Rb at T373, known to completely inactivate Rb, was upregulated in CRC and UCEC tumors. These results indicate both unique and shared Rb phosphosites across cancer types with functional significance.

\begin{figure}[p]
    \centering
    \phantomlabel{fig:ptmcosmos-rb1-prot-paint}
    \phantomlabel{fig:ptmcosmos-rb1-e2f}
    \phantomlabel{fig:ptmcosmos-lscc-ubiquityl}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure5_ptmcosmos_rb1.pdf}
    \caption[Pan-cancer Rb phosphorylation analysis and LSCC ubiquitination analysis.]{%
        Pan-cancer Rb phosphorylation analysis and LSCC ubiquitination analysis.
        \legendcontdnote
    }
    \label{fig:ptmcosmos-rb1}
\end{figure}
\begin{figure}[t]
    \centering
    \legend{%
        \legendcontdref{fig:ptmcosmos-rb1}
        \subref{fig:ptmcosmos-rb1-prot-paint}
        Overview of differentially expressed Rb phosphosites across 8 cancer types and the mapping of these phosphosites onto the domain structure of Rb. The numbers shown in circles represent the number of samples in the given cohort detecting a phosphosite.
        \subref{fig:ptmcosmos-rb1-e2f}
        Averaged expression of E2F target genes (``E2F score'') stratified by \gene{RB1} alteration status or Rb phosphorylation status.
        \subref{fig:ptmcosmos-lscc-ubiquityl}
        Correlation in lung squamous cell carcinoma between normalized ubiquitylsite abundance and cognate protein abundance for cancer driver genes.
    }
\end{figure}

Due to the importance of Rb phosphorylation in regulating the Rb-E2F interaction, we calculated the E2F transcriptional activity score across cancer types based on gene expression and found it was highly correlated with Rb phosphorylation. Rb-T356 phosphorylation was strongly correlated with E2F activity in ccRCC, GBM and LUAD tumors, suggesting a possible role for this site in the Rb-E2F interaction. We also observed site- and cancer type-specific associations between Rb phosphorylation and E2F activity. Phosphorylation of Rb at T373 was significantly correlated with E2F activity in UCEC tumors. Dual phosphorylation at S807/S811 and T823/T826 were also significantly associated with E2F activity in UCEC and GBM tumors. ccRCC tumors showed strong correlations between Rb-T252 phosphorylation and E2F activity.

Next, we investigated the effect of Rb genetic alterations on the relationship between Rb phosphorylation and E2F activity. We divided tumor samples into an Rb altered group, consisting of samples with either \gene{RB1} somatic mutations or copy number loss, and an Rb unaltered group lacking such alterations. Interestingly, unaltered samples showed stronger associations between Rb phosphorylation and E2F activity, including associations not observed when computing correlations across all tumor samples. GBM unaltered tumors showed significant correlations between E2F activity and Rb-S612 phosphorylation, known to disrupt the E2F binding cleft in the Rb pocket domain \cite{macdonaldji_dickfa:PosttranslationalModifications2012}. Significant correlations for Rb-S788 were also observed in GBM unaltered tumors. LUAD unaltered tumors exhibited a significant correlation between E2F activity and Rb-T821/T826 phosphorylation, known to disrupt the RbC-E2F interaction through promoting binding of RbC to the pocket domain of Rb \cite{rubinsm_pavletichnp:StructureRb2005}. Additionally, LUAD unaltered tumors showed a strong association between Rb-T356 phosphorylation and E2F activity, consistent with our observations in ccRCC, GBM and UCEC. These results suggest that 1) cancer types may utilize distinct Rb phosphosites across domains to promote E2F activity and 2) tumors lacking classical \gene{RB1} genetic alterations may utilize Rb phosphorylation to drive E2F activity. We further investigated the interplay of \gene{RB1} genetic alterations and Rb phosphorylation by comparing E2F activity between unaltered and altered samples. As expected, unaltered phospho-high tumors and altered tumors both showed significantly increased E2F activity when compared to unaltered phospho-low tumors across cancer types. Strikingly, no significant difference in E2F activity was observed when comparing unaltered phospho-high tumors and altered tumors, suggesting that tumors without genetic alterations in \gene{RB1} may utilize Rb phosphorylation to achieve similar levels of E2F activity to \gene{RB1} altered tumors.

Additionally, we investigated the possible mechanisms underlying increased Rb phosphorylation across cancer types. First, we used multi-omics data to investigate the correspondence between \gene{RB1} RNA expression, Rb protein abundance, and Rb phosphorylation. As expected, LUAD tumors showed significantly decreased \gene{RB1} RNA expression compared to normal adjacent tissue (NAT). Interestingly, LUAD tumors showed no significant difference in total protein abundance compared to NAT, but significant upregulation of Rb phosphorylation, suggesting that Rb phosphorylation is regulated independently of \gene{RB1} expression and total Rb protein abundance in LUAD. Intriguingly, UCEC and ccRCC tumors showed global upregulation of \gene{RB1} RNA, Rb protein and phosphorylation abundance. Unlike previous studies implicating \gene{RB1} copy number amplification in contributing to increased Rb phosphorylation in tumors, upregulated \gene{RB1} expression did not appear to be due to copy number amplification in these cohorts \cite{vasaikars_zhangz:ProteogenomicAnalysis2019}. These results suggest that LUAD tumors may utilize a distinct mechanism from UCEC and ccRCC tumors in regulating Rb phosphorylation.

To further characterize the regulation of Rb phosphorylation, we investigated the relationship between Rb phosphorylation and cyclin-dependent kinases (Cdk) across cancer types. LUAD tumors showed significant correlations between CDK1 protein abundance and phosphorylation of Rb at S249, T356 and the dual phosphorylation sites T821 and T826, with CDK1 being a putative in vivo kinase of S249 phosphorylation \cite{hasslerm_mittnachts:CrystalStructure2007}. Similarly, in ccRCC tumors, CDK1 protein abundance was strongly correlated with phosphorylation at S249 and T373, both of which are putative CDK1 sites (PSP). ccRCC tumors also showed a strong and significant correlation between Rb-T356 phosphorylation and CDK1 protein abundance, although this phosphosite is not known to be phosphorylated by CDK1. In UCEC tumors, Rb phosphorylation at the putative CDK1/CDK2 sites of S807/S811 and T373 were significantly correlated with both CDK1 and CDK2 protein abundance (PSP). Additionally, phosphorylation at the putative CDK2 phosphosite T356 was strongly associated with CDK2 protein abundance across UCEC tumors. GBM tumors exhibited strong correlations between the protein abundance of the canonical Rb kinase CDK6 and multiple Rb sites, including the known CDK6 site S780 (dually phosphorylated with T778) and the putative CDK6 sites S612, S788, S795, S807, S811 and T826 (dually phosphorylated with T823).  Similar to other cancer types, GBM tumors also showed significant correlations between CDK1 protein abundance and multiple Rb phosphosites, including the dual phosphorylation sites S807/S811 and S249/T252. Our results suggest that 1) in LUAD, UCEC, GBM and ccRCC tumors CDK1 may phosphorylate Rb at distinct sites, 2) CDK2 may additionally regulate Rb phosphorylation in UCEC tumors and 3) CDK6 may additionally phosphorylate Rb in GBM tumors.

Finally, we investigated the association between Rb phosphorylation and clinical and subtype data. For each cancer type, we divided samples into three groups (low, medium and high) based on an aggregated Rb phosphorylation score. GBM tumors showed multiple significant associations between subtype and clinical features and Rb phosphorylation status. High Rb phosphorylation was associated with the im4 immune subtype, consistent with this subtype being enriched in mitotic cell cycle and mitotic spindle checkpoint pathways. Rb phosphorylation was enriched in the nmf3 cluster consisting of classical-like tumors, and the transcriptomic classical subtype, consistent with enrichment of cell cycle pathways in this cluster. Interestingly, low Rb phosphorylation was associated with increased telomere length. Recently, inhibition of the glycoprotein MUC1 in GBM cell lines was found to decrease phosphorylation of Rb (at S780) and promote alternative lengthening of telomeres, consistent with the trend we observed \cite{kims_parkck:InhibitionMUC12020}. In addition, for each cohort of interest, we investigated the relationship between Rb phosphorylation and significantly mutated genes as defined by each large-scale proteogenomic study \cite{douy_zhaog:CPTACUCEC2020,gillettema_shiz:ProteogenomicCharacterization2020,wanglb_cptac:GBM2021}. In UCEC tumors, genetic alterations in the chromatin-binding protein CTCF were associated with high Rb phosphorylation, consistent with CTCF binding to the \gene{RB1} promoter to prevent epigenetic silencing \cite{rosa-velazqueziadl_recillas-targaf:EpigeneticRegulation2007}. In LUAD tumors, \gene{RB1} mutations were associated with low Rb phosphorylation, suggesting that samples possessing mutations in \gene{RB1} may not be dependent on Rb phosphorylation. LUAD tumors also showed a significant association between mutations in both \gene{STK11} and \gene{KEAP1} and high Rb phosphorylation, consistent with previous studies showing that alterations in these tumor suppressor genes are mutually exclusive with \gene{RB1} alterations in neuroendocrine lung tumors \cite{georgej_thomasrk:IntegrativeGenomic2018}. GBM tumors showed a similar association between \gene{RB1} genetic alterations and low Rb phosphorylation, with TP53 genetic alterations also correlating with low Rb phosphorylation in this cohort. Thus, our analyses connect Rb phosphorylation with subtype differences and significantly mutated genes across cohorts.

\begin{figure}[p]
    \centering
    \phantomlabel{fig:ptmcosmos-hotptm-enrich}
    \phantomlabel{fig:ptmcosmos-hotptm-structure}
    \phantomlabel{fig:ptmcosmos-hotptm-structure-complex}
    \phantomlabel{fig:ptmcosmos-hotptm-pi3k}
    \includegraphics[width=\linewidth]{figures/chap03_ptmcosmos/figure6_ptmcosmos_hotptm.pdf}
    \caption[Spatial co-clustering of mutations and PTMs.]{%
        Spatial co-clustering of mutations and PTMs.
        \legendcontdnote
    }
    \label{fig:ptmcosmos-hotptm}
\end{figure}
\begin{figure}[t]
    \centering
    \legend{%
        \legendcontdref{fig:ptmcosmos-hotptm}
        \subref{fig:ptmcosmos-hotptm-enrich}
        Enrichment of activating mutations among mutations co-clustering with PTMs.
        \subref{fig:ptmcosmos-hotptm-structure}
        Selected mutation-PTM clusters mapped onto crystal structures obtained from the Protein Data Bank. Red spheres represent the residues modified by PTM sites, while blue spheres represent the residues affected by mutations.
        \subref{fig:ptmcosmos-hotptm-structure-complex}
        Visualization of mutation-PTM clusters spanning protein complexes. Purple spheres represent residues affected by both PTM sites and mutations.
        \subref{fig:ptmcosmos-hotptm-pi3k}
        Relative abundance of the PIK3R1-Y452 phosphosite in BRCA and UCEC stratified by mutation co-clustering status.
    }
\end{figure}

\section{Discussion}



\section{Methods}

\subsection{Data sources}
\subsubsection{CPTAC proteomic data processing}

\tref{tab:ptmcosmos-peptide-db} \url{https://github.com/ccwang002/cptac_proteome_preprocess}


\begin{table}[tbp]
    \centering
    \caption{CPTAC peptide search databases used by different disease working group.}
    \label{tab:ptmcosmos-peptide-db}
    \begin{threeparttable}[b]
    \begin{tabular}{@{}llll@{}}
    \toprule
    Phase & Cohort & Database & Notes \\
    \midrule
    \multirow{2}{*}{\begin{tabular}[c]{@{}l@{}}CPTAC2/TCGA\\ retrospective\end{tabular}}
        & BRCA  & RefSeq 20130727   & Broad Institute \\
        & OV    & RefSeq 20111201   & PNNL \\
    \midrule
    \multirow{3}{*}{\begin{tabular}[c]{@{}l@{}}CPTAC2\\ prospective\end{tabular}}
        & OV    & CDAP (Refseq 2016)    & PNNL \\
        & CRC   & RefSeq 20171003\tnote{*}  & Use RefSeq 2018 instead \\
        & BRCA  & CDAP (Refseq 2016)    & Broad Institute \\
    \midrule
    \multirow{8}{*}{\begin{tabular}[c]{@{}l@{}}CPTAC3\\ discovery\end{tabular}}
        & CCRCC & CDAP (Refseq 2018) & UMich \\
        & GBM   & CDAP (Refseq 2018) & PNNL \\
        & HNSCC & CDAP (Refseq 2018) & UMich \\
        & LUAD  & CDAP (Refseq 2018)\tnote{\textdagger} & Broad Institute \\
        & LSCC  & CDAP (Refseq 2018)\tnote{\textdagger} & Broad Institute \\
        & PBTA  & CDAP (Refseq 2018) & MS3 spectra \\
        & PDAC  & CDAP (Refseq 2018) & UMich \\
        & UCEC  & CDAP (Refseq 2018) & PNNL \\
    \bottomrule
    \end{tabular}
    \begin{tablenotes}
    \item [*] Database uses hg38 genome reference.
    \item [\textdagger] Database includes smORFs.
    \end{tablenotes}
    \end{threeparttable}
\end{table}


\subsubsection{PTM coordinate harmonization and protein sequence alignment}
Only considered the canonical reviewed UniProt entries 
If the identical protein sequence can be found, mapped to them directly
Otherwise, perform global protein sequence alignment
To the UniProt entry of the same gene
Create coordinate segments of consecutive matches (don't allow any amino acid mismatch)
Only mapped the site when a full peptide (plus the 1 additional flanking aa) is in the coordinate segment

\subsection{Database and website development}
