\chapter{Conclusion and Future Directions}
\label{chap:conclusion}
% Discussion: remaining challenges, the benefits in another fields than oncology

My dissertation contributes to a better toolbox for cancer proteogenomic and multi-omics characterization and generates insights into glioblastoma disease biology that will lead to novel therapeutic avenues. In \Cref{chap:mut-pipeline-qc}, we investigated the data harmonization using standardized data processing pipelines on establish bulk genomic sequencing technologies. Data harmonization is part of the critical infrastructure of large-scale multi-omics studies to ensure the generated datasets can be integrated with other data of interest and re-used by the research community. We showed that the data harmonization on Genomic Data Commons is applicable to various studies across cancer types. More importantly, the harmonized outputs do not significantly alter the downstream biological findings and interpretation. In the cases where technical artifacts are introduced, those artifacts are carefully characterized and documented. Pipelines on GDC have since been applied to more cancer projects. Other data commons of different data types are actively being developed, including Proteomic Data Commmons and Imaging Data Commons, by following the example set by GDC.

In \Cref{chap:ptmcosmos}, we developed a database, PTMcosmos, that collects the existing knowledge of post-translation modifications in cancer and used the integrated information of PTmcosmos to analyze the experiment datasets from Clinical Proteomic Tumor Analysis Consortium to showcase its potential values to the research community. We collected the proteomic datasets from CPTAC, a consoritum with the goal of proteogenomic multi-omics characterization of patient tumors across multiple cancer types. By harmonizing the CPTAC's mass spectrometry based global protein and PTM abundance measurements with existing supporting evidences of PTM sites, we were able to identify pan-cancer PTM dysregulation events that are not explained by the known upstream genetic alterations or associated with clinically relevant tumor subtypes. We also investigated the mutational impact on PTM through protein structure guided spatial clustering. Furthermore, the easy-to-use user interface of PTMcosmos brings such functionalities to the end user, allowing the broader research community to utilize the immense amount of new PTM information brought by CPTAC and the known functions and annotations of PTMs from the literature.

In \Cref{chap:cptac-gbm-discov}, we focused on the proteogenomic characterization of glioblastoma in a CPTAC study. During the data analysis, we utilized the ``toolbox'', where the genomic data were harmonized on GDC and PTM annotations were pulled from PTMcosmos. Our analysis contributes to the potential improvement of patient stratification and discovery of new therapeutic options. Using multi-omics clustering, we identified a subset of patients with mixed subtypes compared with traditional sequencing-based subtypes, who exhibit shortened overall survival. Phosphoproteomic data indicate that PLCG1 and PTPN11 act as a common signaling hub for multiple RTKs. Given the high RTK genetic alteration frequencies and eventual remission in GBM, a combined therapy targeting both RTKs and their shared signaling hub might be more effective. Using connectivity map approach, we identified some druggable targets based on the gene and protein signatures of GBM tumors. By integrating the bulk multi-omics with single cell transcriptomics, we dissected the tumor microenvironment and characterized the immune composition differences and the potential signaling transduction regulation. In sum, our findings bring new insights towards a better understanding of GBM and hopefully a better personalized treatment for GBM patients.

Undoubtedly, there are new set of challenges that call for an upgraded bioinformatics toolbox and better multi-omics integration. The function of many PTM sites remain unknown and there is a lack of systematic understanding of the PTM regulation across cancer types. GBM remains difficult to treat and the mechanism behind the eventual recurrence of the tumor is elusive. We will dive into these challenges and future directions below.


\section{New technologies call for new harmonization pipelines and data repositories}
In the past decade, the single cell technologies have made great breakthroughs that they are more accessible for large scale multi-omics studies \cite{chappelll_voett:SingleCellMulti2018}. Particular for single cell transcriptomics technologies, it is now possible to profile the full gene expression of tens of thousands of human cells in a single cell, generating high dimension cell-by-gene expression matrices that take more spaces to store and more computational power to process. While the existing data repositories like GDC are able to host the single cell data, there has not been a standard way to let users explore and extract part of the datasets for their down downstream analysis. On the other hand, the bulk genomic datasets can be explored more easily through integration portals like cBioPortal \cite{ceramie_schultzn:CBioCancer2012,gaoj_schultzn:IntegrativeAnalysis2013}. Due to the high dimensionality of the single cell data (cell per sample per cohort), new innovations are required to redesign the user interface and visualization. More broadly speaking, new imaging based/assisted single cell technologies such as multiplexed imaging \cite{giesenc_bodenmillerb:HighlyMultiplexed2014,goltsevy_nolangp:DeepProfiling2018,tanwcc_limtkh:OverviewMultiplex2020} and spatial transcriptomics \cite{morrissa_morrissa:PinpointingSpatial2019,longosk_khavaripa:IntegratingSinglecell2021,driesr_yuangc:AdvancesSpatial2021} will bring in a new set of challenges to integrate the genomic data with imaging data.

Like bulk genomics data, data generated by the new technologies also requires a careful review and quality check of their data harmonization. For single cell transcriptome technologies, the data processing and integration methods are still under active development and lack consensus \cite{lahnemannd_schonhutha:ElevenGrand2020,andrewsts_hembergm:TutorialGuidelines2021}. As we consistently utilize new omics datasets, the same set of challenges will appear and require similar data quality control as described in \Cref{chap:mut-pipeline-qc}.


\section{Adoption of universal gene annotation with sequence identity}
Encountered in \Cref{chap:mut-pipeline-qc} and \ref{chap:ptmcosmos}, the choice of the identifier system and the cross reference between different gene annotation systmes and even different version of the same annotation system can be problematic, causing either ambiguous or incompatible mapping. At the first step towards the consistent annotation, all multi-omics datasets should keep track of the annotation and version in use and ensure the sequence stability of the annotation when necessary, as described in \Cref{chap:intro}. With the correctly documented annotations, we should aim to use the same annotation across different omics data, which will greatly reduce the unnecessary technical issues incurred during the data integration. However, as new annotations are released regularly, we will eventually face the problem of combining datasets of different annotations and this problems have become more apparent to the community decades into the cancer genomics era. Unfortunately, we do not have a comprehensive solution towards integration of different annotations yet. It is currently recommended to re-process or re-annotate the data to complete get rid of the problem.

There are some ongoing efforts towards an optimal solution of annotation cross reference with sequence identity at transcript level. The Matched Annotation from NCBI and EMBL-EBI (MANE) project (\url{https://www.ncbi.nlm.nih.gov/refseq/MANE/}) is a collaboration of RefSeq and Ensembl, the two most commonly used annotation systems, to unify the transcript models of human genes. The goal of MANE is to provide a set of matching RefSeq and Ensembl transcripts of protein-coding genes, where the matched transcripts have identical sequences, including 5' UTR, coding region and 3' UTR. The latest version (v0.95 released on 2021-06-16) of MANE covers over 95\% of the human protein-coding genes, where every gene has at least one matching transcript. This is an exciting milestone since most of the daily analysis and disease relevant genes can be covered and we start to see more bioinformatic tools prioritize the matched annotations and gene models. This will greatly reduce the incompatibility in the annotation cross reference between RefSeq and Ensembl. The Ensembl Transcript Archive (Ensembl Tark; \url{https://tark.ensembl.org/}) tackles the probelm from a different angle. By archiving all the transcript sequences and their gene models across all Ensembl and RefSeq releases, Tark is able to systematically compare the gene model changes across any two different transcripts of the same gene. While it doesn't solve the sequence identity mapping problem, it is hugely beneficial that users can now easily select compatible sets of annotations. For example, protein based analyses only consider the identity of the protein coding sequencing, so more transcripts will be compatible even if their UTR sequences are unmatched.

At the protein level, UniParc \cite{leinonenr_apweilerr:UniProtArchive2004} provides a comprehensive collection of all recorded protein sequences and their annotations to external databases, which can also be used to check for sequence identity of two proteins.
However, it lacks the interface to systematically check all the proteins of one annotation sources and does not show the sequence difference for two unmatched proteins. Future work should consider a system similar to MANE or Ensembl Tark to allow users to easily map the proteins across different annotation systems. The coordinating mapping algorithm developed by PTMcosmos for PTM sites can be generalized as a standalone application for protein cross reference.



\section{Pan-cancer PTM study with experimental validation}
PTMcosmos provides a resource to study PTM function in cancer. Due to the usage of datasets from the original publication, it limited our ability to conduct a pan-cancer PTM abundance comparison. The normalized abundance values can only be compared within each study. A reprocessing of all CPTAC proteomics datasets is required to obtain the comparable normalized values. The data reprocessing, or a better data harmonization, will enable us to unlock many possible future analyses. For example, quantitative analyses on the abundance level change will be feasible; a pan-cancer multi-omics clustering will allow us to identify shared activated oncogenic progress across cancer types, including phenotypes such as the cell proliferation, DNA damage repair, and immune infiltration. With the pan-cancer CPTAC data reprocessing, we are also able to investigate the PTM crosstalks between phosphorylation, acetylation, and ubiquitylation. The current peptide spectra search does not allow peptides with multiple PTM types. It is unclear whether it is technically possible to integrate the phospho-, acetyl-, and ubiquityl-enriched spectra together to infer the proportion of peptides with PTM sites of multiple types coexisting. These directions are actively explored in the CPTAC pan-cancer working groups currently.

A future challenge for proteomic study is to identify the enzymes regulating the individual PTM sites. CPTAC reported many more PTM sites of unknown function and unknown regulator. For phosphorylation, since the kinase and substrate specificity is associated to the substrate sequence and local structure, some prediction tools are developed based on the known substrate motifs of the kinases. Through experimental screening of the substrate sequence preference of the human kinases (e.g, 5 to 10 amino acids flanking sequences of a phosphosite), we might be able to construct the specificity model and predict the upstream regulating kinases of a PTM site. Such modeling will greatly advance our understanding on the phosphosites of unknown function. Similar screening methodology can be applied to other PTM types and PTM crosstalks to identify their substrate sequence preference.


\section{Next steps for CPTAC glioblastoma study}

\begin{figure}[tb]
    \centering
    \includegraphics[width=0.6\linewidth]{figures/chap05_conclusion/cptac_gbm_cancer_cell_cover.png}
    \caption[Better disease understanding through a lens for an integrative view of multi-omics datasets.]{Better disease understanding through a lens for an integrative view of multi-omics datasets. Cover art of \textit{Cancer Cell} (April 2021 issue). Artwork by Jessica Johnson \url{https://jessicajohnsonart.com/}.}
    \label{fig:lens-multi-omics}
\end{figure}

Our proteogenomic characterization provides a glimpse into the better disease understanding and personalized medicine using multi-omics integration (\fref{fig:lens-multi-omics}). Per the design of CPTAC studies, an immediate follow up cohort (confirmatory cohort) is undergone to validate the findings in the publish discovery cohort. In the confirmatory cohort, targeted mass spectrometry panels are designed to include the selected protein and phosphopeptides of interests that will greatly improve the detection and quantitative accuracy of abundance measurement.


\begin{figure}[tb]
    \centering
    \phantomlabel{fig:cptac-gbm-future-plan-longitudinal}
    \phantomlabel{fig:cptac-gbm-future-plan-heterogeneity}
    \includegraphics[width=1\linewidth]{figures/chap05_conclusion/gbm_future_plan.pdf}
    \caption[CPTAC GBM study future plan.]{%
        CPTAC GBM study future plan.
        \subref{fig:cptac-gbm-future-plan-longitudinal} Longitudinal collection of tumor samples and paired normal samples.
        \subref{fig:cptac-gbm-future-plan-heterogeneity} Tumor heterogeneity using single cell technologies.
        \sourceatright[2em]{\footnotesize Created with BioRender.com}
    }
    \label{fig:cptac-gbm-future-plan}
\end{figure}

To advance our understanding of the glioblastoma disease progression, we also revised our confirmatory cohort study design (\fref{fig:cptac-gbm-future-plan}). We included the WashU Legacy cohort, where matched normal samples at the mirror side of the brain were collected from the GBM patients. Since most of the GBM patients reoccur eventually after treatment, we included a retrospective longitudinal collection from CHOP (Children's Hospital of Philadelphia) and performed the proteogenomic characterization of the collected primary and recurrent tumors. Regarding the multi-omics assays for this cohort, on top of the existing assays, we expanded the single cell experiments to conducted single nuclei RNA sequencing on more samples. The expansion of the sample inclusion allow us to study the tumor microenvironment and tumor clonal evolution in \gene{IDH} mutated tumors and primary and recurrent tumor pairs. We also obtained the single cell chromatin accessibility profiles on the selected samples using single nuclei ATAC sequencing. For some of the sample, a newer single cell multiome assay was used to capture the gene expression and chromatin accessibility simultaneously in the same cell, allowing to directly infer the transcriptional regulation on the genomic regulatory elements on gene expression.

There are a few future analysis directions that can be answered using our datasets and contributed to the field. We proposed a multi-omics classifier that takes in CNV, RNA, protein, and phospho bulk features, however, we have not fully explored the multi-omics classification that include additional features such as acetylation, DNA methylation and other data types due to the limited sample size and the increasing computational complexity. We will explore the multi-omics classifications with more data types and assess if they are applicable to recurrent tumors. On the other hand, the current multi-omics classifier requires thousands of features across data types, making it be difficult to implement in a more clinical setting and thus limits its application. We would like to develop a version of the classification with similar performance but uses much fewer features. During the process of limiting our features, we will also identify the most representative features to each multi-omics subtypes.

In the discovery study, we show that many GBM subtype features are driven by the cell type composition difference in the tumor microenvironment. To identify the tumor intrinsic subtypes, we will utilize the increase number of samples with single cell data. In a straightforward direction, we can investigate the relationship between tumor intrinsic subtypes, the tumor microenvironment differences, and other clinically relevant phenotypes. While single cell transcriptome driven GBM subtyping exists \cite{suvaml_tiroshi:GliomaStem2020,pinear_fineha:TumorMicroenvironment2020}, we want to integrate the gene expression with the differences in the chromatin accessibility lanscape using snATAC-seq data. The transcription regulatory network is closely tied with the regulation of transcription factors and thus cell type dependent. By investigating the chromatin accessibility differences, we can also infer the potential cell of origins of GBM tumors. Moreover, G-CIMP phenotype and DNA methylation subtypes indicate the epigenomic regulation of the gene expression in GBM. Thus, in addition to the known association of the histone acetylation with DNA methylation we observed in our study, we want to see if they both lead to the change in chromatin accessibility and identify the regulated gene loci.

Druggability prediction from the disease gene and protein signatures is a powerful way to explore the novel avenues for GBM treatment. We explore the genetic alteration based signatures such as \gene{EGFR} and \gene{RB1} in the study, however, the same method can also be directly to the multi-omics subtype signatures. The druggability prediction reports compounds enriched with certain kinds of mechanism of action (MoA). We can follow up on the identified MoAs using pathway analysis to further select the compounds of interest that showed increased activity in the corresponding pathways. Finally, with the single cell dataset, we can predict the tumor intrinsic druggable targets by extracting the signatures of only the tumor cells. As the signatures are matched again cell line based perturbation datasets, tumor intrinsic signatures might be more applicable due to the isolation from the tumor microenvironment. On the other hand, if we are able to obtain the perturbation datasets using immune cells (e.g., myeloid cells), another possibility is to identify the druggable compounds that target the tumor associated macrophages observed in the more aggressive mesenchymal GBM tumors.
